gitapp/lib/user_details/user_details.dart

```diff
@@ -13,9 +13,73 @@ class DetailScreen extends StatelessWidget {
       appBar: AppBar(
         title: Text("User details"),
       ),
-      body: Padding(
-        padding: EdgeInsets.all(16.0),
-        child: Text("User details screen"),
+      body: Container(
```

Comment:
Maybe we could extract this widget to single class ??

Reply:
not mine